ct/homarr.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
 # Copyright (c) 2021-2024 tteck
-# Author: tteck (tteckster)
+# Author: MickLesk (Canbiz)
```

Comment:
I would leave tteck as the original author and add yourself as a revision author, maybe. 🤔